ctapipe/resources/base_config.yaml

```diff
@@ -95,7 +95,9 @@ ImageProcessor:
 ShowerProcessor:
   # choose between HillasReconstructor and HillasIntersection or both (two
   # implementations of the standard stereo line-intersection method)
-  reconstructor_types: [HillasReconstructor, HillasIntersection]
+  reconstructor_types:
+    - HillasReconstructor
+      #  - HillasIntersection
```

Comment:
Why commented out?

Reply:
It was recommended by @kosack 
> so perhaps comment-out the HillasIntersection line just to show that it can be enabled, but is not enabled by default

Reply:
Ah ok, good then